WMass/python/plotter/tree2yield.py

```diff
@@ -15,16 +15,9 @@
 
 from copy import *
 
-from CMGTools.WMass.plotter.cutsFile import *
```

Comment:
can you leave this also here? This is integrating part of tree2yield even if one would not use fakeRate.py